manifests/opendatahub/base/kustomization.yaml

```diff
@@ -3,30 +3,22 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  # Build Configs
-  - ./buildconfigs/visualization-server.yaml
-
   # CustomResourceDefinitions
-  - ./customresourcedefinitions/viewers.yaml
   - ./customresourcedefinitions/scheduledworkflows.yaml
 
   # Deployments
   - ./deployments/ds-pipeline-persistenceagent.yaml
   - ./deployments/ds-pipeline-scheduledworkflow.yaml
-  - ./deployments/ds-pipeline-viewer-crd.yaml
```

Comment:
how come we're removing viewer-crd controller as well? I'm not sure I even know what this does, does it not make sense to have this if we don't have visualization server?

Reply:
Visualization Server, along with Viewer CRD, composes the feature of pipeline outputs w/ visualizations included. While Viz Server is the runtime for visualizing some of the visualizations, Viewer CRD watches for Viewers to deploy the generated visualizations in a separate pod.